### ISSUE TITLE
Rank1D graph has an argument `color`. But when we pass a color as a string its simply not working.

### ISSUE THREAD

**Issue by sanu-s** [DistrictDataLabs/yellowbrick#964]
**Describe the bug**
I'm using yellow brick version 0.9.1. I couldn't change the colour of Rank1D bar digram. It's set to blue by default and not changing

**To Reproduce**

```python
from yellowbrick.features import Rank1D

# Load the credit dataset

# Instantiate the 1D visualizer with the Sharpiro ranking algorithm
visualizer = Rank1D(algorithm='shapiro', color='red')

visualizer.fit(X, y)           # Fit the data to the visualizer
visualizer.transform(X)        # Transform the data
visualizer.poof()   
```

**Dataset**
I used the classical diabetes data

**Expected behavior**
I would like to change the colour of Rank2D graph from blue to red, green or any

**Traceback**

```
If applicable, add the traceback from the exception.
```

**Desktop (please complete the following information):**
 - OS: [Ubuntu 18.04.1]
 - Python Version [3.6]
 - Yellowbrick Version [e.g. 0.9]

**Additional context**

**Comment by lwgray** [comment]
Hi @sanu-s Thanks for using Yellowbrick.  I am not able to reproduce your problem.  Can you provide more info (see above template questions) ?

**Comment by sanu-s** [comment]
Hi @lwgray Thanks for the response. I have updated the question using template

**Comment by lwgray** [comment]
@sanu-s I am still not able to reproduce the bug (see below) . Are you using the latest version of Yellowbrick?  pip install yellowbrick --upgrade

Do @bbengfort @rebeccabilbro or @Kautumn06 have any suggestions?

```
from sklearn.datasets import load_diabetes
from yellowbrick.features import Rank1D


# Load the diabetes dataset
diabetes = load_diabetes()
X, y = diabetes.data, diabetes.target

for color in ['red', 'green', 'blue']:
    # Instantiate the 1D visualizer with the Sharpiro ranking algorithm
    visualizer = Rank1D(algorithm='shapiro', color=color)

    visualizer.fit(X, y)           # Fit the data to the visualizer
    visualizer.transform(X)        # Transform the data
    visualizer.poof() 
```

![image](https://user-images.githubusercontent.com/1000117/64386514-53c0bf00-d007-11e9-9139-4438deb92369.png)

![image](https://user-images.githubusercontent.com/1000117/64386525-5d4a2700-d007-11e9-957e-3a0e3127c4f2.png)

![image](https://user-images.githubusercontent.com/1000117/64386541-6a671600-d007-11e9-8c36-7bf55c03dcbe.png)

![image](https://user-images.githubusercontent.com/1000117/64386575-85398a80-d007-11e9-8b6c-75f2294c61d8.png)

**Comment by lwgray** [comment]
@sanu-s I just got some feedback from yellowbrick maintainer/founder @rebeccabilbro .   She suggests upgrading to version 1. She added the color param to rank1d and a bunch of the other barcharts in the v1.0 release

**Comment by sanu-s** [comment]
@lwgray the issue seems fixed in v1.0 thanks you so much
